pkgs/misc/rivalcfg/default.nix

```diff
@@ -0,0 +1,19 @@
+{ stdenv, fetchurl, python3Packages }:
+
+with python3Packages; buildPythonPackage rec {
+  pname = "rivalcfg";
+  version = "3.6.0";
+
+  meta = with stdenv.lib; {
+    description = "Small CLI utility program that allows you to configure SteelSeries Rival gaming mice";
+    license     = licenses.wtfpl;
+  };
+
+  src = fetchurl {
+    url    = "https://github.com/flozz/rivalcfg/archive/v${version}.tar.gz";
+    sha256 = "78bd2ed860ad62b5bcd2fba334d8b9fda04524f7d45dd4f57dc20d9b4124110b";
+  };
+
+  doCheck = false;
```

Comment:
do the tests not work?

Reply:
They don't, it won't install with them enabled.

Reply:
does it look fixable? it would be better to have the tests if they can be made to work

Reply:
Probably not by me, I looked into it a bit when it was failing and had no idea what it was doing.